### DISCUSSION TITLE
2 suggestions and a thank you

### DISCUSSION THREAD

**Discussion by Gaerthal-Estelmist** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#71] [Ideas]
Hey, I just wanted to thank you again for your continued support for this amazing project. After seeing my issue closed, I tried the latest patcher version and finally it works perfectly again. No longer CTD or FPS drops. Happiness!

Two small improvement suggestions that I've had on my mind for a long time:
1- When the patcher GUI is opened, it should automatically detect the Sims.exe path and writes it in the Browse section.
2- When the mouse is held over Resolution drop-down, an info pops-up and says that 800x600 and 1024x768 are the two default resolutions for The Sims. In addition to that info, the list itself should show which options adhere to the native game resolutions. Meaning, there should be an indicator (like "Default", "Native", "Original Aspect Ratio" or such ) next to these 1.3 ratio resolutions: 640×480, 800×600, 960×720, 1024×768, 1280×960, 1400×1050, 1440×1080 , 1600×1200, 1856×1392, 1920×1440, and 2048×1536. Would make it easier to decide, I guess.

**Comment by FaithBeam** [comment]
Thanks for sticking around, and I'm happy this release solved that outstanding issue for you. Those are good suggestions. #&#8203;1 will be included in the next release for sure. #&#8203;2 is somewhat difficult to code and to figure out how I want to present that information. Maybe I color code the resolutions based off their aspect ratios, or I add another box that ratios can be selected to filter the resolution list with.

**Reply by Gaerthal-Estelmist** [reply]
I also thought of color coding a few hours after I posted it, but did not comment again. Nice to see you thought of it as well. Either way would be nice. Maybe native resolutions can be in green, the ones you deem "having graphical issues" in red, the rest the default color. And the hover-over info includes information about which color represents what.

**Reply by FaithBeam** [reply]
I finally got around to this suggestion. Sorry for the delay. Let me know your thoughts on the pull request #76 
Thanks!

**Reply by Gaerthal-Estelmist** [reply]
No worries for the delay, I'm always happy to see your entries, whenever they come :)
I checked the entry, and everything you listed there, the screenshots as well, look great! It is even better than what I had in mind and I could think of nothing to add to them. The only thing is, my first suggestion (automatic path) is not listed there. Do you plan to add it later?
I'm somewhat inexperienced when it comes to github processes. Can you please tell me what I need to do in a pull request? Do I need to do anything in that entry? It says it needs a review, but I'm not sure if I'm the one who is supposed to do that.
Also, I could not find how I could try / test your updates. Is there a way to download a release and test them, or is the pull request simply there as a preview to discuss about?
Thanks again!

**Reply by FaithBeam** [reply]
Actually, I snuck the feature of finding the Sims installation path in #72 which is version 3.6.0, but I didn't formally release it. It's a personal preference of mine to not release new versions of the patcher quickly unless there's a killer feature, or if enough features have built up. 

You don't need to do anything with the pull request, that review is for me to make sure I look over the changes for any obvious issues. I don't know if you're able to download the artifacts from the automatic build. Are you able to download the windows artifact here? https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/actions/runs/6447223864#artifacts

![image](https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/assets/32502411/8f42d1ee-5559-454c-bf76-e319307509af)

I can't attach the zip because there's a 25mb limit and it is 37mb. If you can't download from the artifacts section, I'll probably release it as a beta to the normal Releases page.

**Comment by Gaerthal-Estelmist** [comment]
Appreciate the explanations.
I could indeed download that entry and test the patcher. Everything seems to work A-Okay :)
I also read your note about how this improvement was a challenge. I'm sure it took much time but in the end, your knowledge is now richer. And we got a sweeter patcher. I'd say it is a win-win :)
Avast One tried to block me from using it, saying it should be tested first (in their lab, whatever that means) and it could take a few hours. Of course I used it anyway, knowing it is an unnecessary precaution when it comes to your work.
Keep up the awesomeness!

**Reply by FaithBeam** [reply]
Awesome! Thanks for the seal of approval and the support over the years. It makes a difference. :)
